### PR TITLE
Update windows readme pthreads

### PR DESCRIPTION
**Pull request by jdelapla**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,14 @@ You can pass the following options to `cmake ..`.
 * `-DUNDEFINED_BEHAVIOR_SANITIZER` -- Build with UndefinedBehaviorSanitizer
 * `-DLINK_PROFILER` -- Link with gperftools (available profiler options are listed [here](https://github.com/gperftools/gperftools))
 
+For windows builds, you will have to include additional flags for libwebsockets CMake. Add the following flags to your cmake command, or edit the CMake file in ./CMake/Dependencies/libwebsockets-CMakeLists.txt with the followig:
```
Comment (disa6302) [inline README.md:100]: Nit: Incorrect spelling of following.

**Review by disa6302 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jdelapla) [reply README.md:100]: correct in one location and my finger skipped an 'n' in another. Fixing.

Comment (hassanctech) [reply README.md:100]: Nit: `w` in windows should be capital.... not blocking

**Review by hassanctech — APPROVED** [review]

**Comment by hassanctech** [comment]
This can be merged since all tests passed except Windows.

**Pull request merged**